src/Gui/InputField.cpp

```diff
@@ -246,7 +246,7 @@ void InputField::newInput(const QString & text)
 
             std::unique_ptr<Expression> evalRes(getExpression()->eval());
 
-            NumberExpression * value = freecad_dynamic_cast<NumberExpression>(evalRes.get());
+            auto value = freecad_dynamic_cast<NumberExpression>(evalRes.get());

```

Comment:
And here (obviously this is nitpicking, if you disagree then just leave it).

Reply:
grrr, ok